register/ABIs/ERC20Detailed.json

```diff
@@ -0,0 +1,267 @@
+[
```

Comment:
Not that these ABI files change much but i'd recommend we actually reference the originals and codegen the bindings using go-ethereum so they always stay in sync. I do that here in gaudius. You can basically copy these two lines.
https://github.com/alecsavvy/gaudius/blob/main/Makefile#L13-L14